### PR TITLE
BUG-102824 Implement Service layer for Filesystem configuration

### PR DESCRIPTION
**Pull request by doktoric**

### CODE REVIEW

File: blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java [hunk 1]
```diff
@@ -31,6 +31,10 @@
         return scripts;
     }
 
+    protected List<FileSystemScriptConfig> getScriptConfigs(Credential credential, T fsConfig) {
```
Comment (mhmxs) [inline blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java]: `T fsConfig` is not in used anywhere

File: blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java [hunk 2]
```diff
@@ -31,6 +31,10 @@
...
+        return new ArrayList<>();
```
Comment (mhmxs) [inline blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java]: `Collections.emptyList()` would be cheaper

File: blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java (hunk 1 above)
Comment (doktoric) [reply blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java]: GcsFileSystemConfigurator is using it

Comment (mhmxs) [reply blueprint-manager/src/main/java/com/sequenceiq/cloudbreak/blueprint/filesystem/AbstractFileSystemConfigurator.java]:
```
echo "Map<String, String> properties = Collections.singletonMap("P12KEY", getPrivateKey(credential));                                                           [master]
        List<FileSystemScriptConfig> fsScriptConfigs = new ArrayList<>();
        fsScriptConfigs.add(new FileSystemScriptConfig("scripts/gcs-p12.sh", POST_AMBARI_START, ALL_NODES, properties));
        fsScriptConfigs.add(new FileSystemScriptConfig("scripts/gcs-connector-local.sh", POST_AMBARI_START, ALL_NODES));
        fsScriptConfigs.add(new FileSystemScriptConfig("scripts/gcs-connector-hdfs.sh", POST_CLUSTER_INSTALL, ONE_NODE));
        return fsScriptConfigs;" | grep "fsConfig." | wc -l
       0
```

**Pull request merged**